### DISCUSSION TITLE
ResetIndexOperation(indexName) does not seem to do anything

### DISCUSSION THREAD

**Discussion by rallman** [ravendb/ravendb#13513] [Q&A]
I have an index that I want to force a reset using the Client API (c#). 
https://ravendb.net/docs/article-page/5.3/csharp/client-api/operations/maintenance/indexes/reset-index

After running the following:
`store.Maintenance.Send(new ResetIndexOperation("MyIndex/Index"));`
Nothing happens on the server. If I feed in an index name that doesn't exist, I get an exception that the index was not found. 

The index takes a few minutes to rebuild, so it's not a question of not seeing the reset fast enough.

I'm running 5.3.2 on the client and server.

**Comment by ayende** [comment]
I double checked the code, and it seems like it certain _should_ work.
If you do the same from the studio, do you observe the same behavior?

Also, why are you resetting the index?

**Reply by rallman** [reply]
I had made a minor change to an index and started getting a bunch of "Could not convert document to entity" exceptions. It seemed that resetting the index from the studio resolved the issue. Turns out, I had an error in my index relating to nulls.
